src/main/java/net/dv8tion/jda/core/events/message/MessageBulkDeleteEvent.java

```diff
@@ -46,11 +46,6 @@ public TextChannel getChannel()
         return channel;
     }
 
-    public Guild getGuild()
```

Comment:
Do you realize that this event doesn't extend the GenericMessageEvent?

Reply:
Oh whoops, reverted